explorer/src/styles.css

```diff
@@ -29,5 +29,12 @@ div div.MuiDataGrid-root .MuiDataGrid-columnHeaderTitleContainer {
 /* Unable to style nested MUI paper within the Drawer
 so styling manually here */
 div.MuiDrawer-root > .MuiDrawer-paperAnchorLeft {
-    background-color: #111826;
+    background-color: #242C3D;
```

Comment:
Try paper props on the Drawer - https://mui.com/api/drawer/

Reply:
Got some TS issues but actually managed to get around them by using nested sx styling. Happy to re-explore if you think its necessary...

Reply:
Sorted.